### PR TITLE
New syntax for more handlers

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR updates a handful of handlers to the new syntax.

### CODE REVIEW

File: skyportal/tests/api/test_thumbnail.py [hunk 1]
```diff
@@ -25,6 +26,16 @@ def test_token_user_post_get_thumbnail(upload_data_token, public_group, ztf_came
     assert status == 200
     assert data['data']['id'] == obj_id
 
+    # wait for the thumbnails to populate
+    nretries = 0
+    thumbnails_loaded = False
+    while not thumbnails_loaded and nretries < 5:
+        thumbnails = DBSession.query(Obj).filter(Obj.id == obj_id).first().thumbnails
```
Comment (guynir42) [inline skyportal/tests/api/test_thumbnail.py:33]: May as well do this with `select` and a context manager?

**Review by guynir42 — APPROVED** [review]
TL;DR
I mean, LGTM. 
One minor comment on the test.

File: skyportal/tests/api/test_thumbnail.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/tests/api/test_thumbnail.py]: yup

File: skyportal/handlers/api/acls.py [hunk 2]
```diff
@@ -26,8 +26,9 @@ def get(self):
                             type: string
                           description: List of all ACL IDs.
         """
-        self.verify_and_commit()
-        return self.success(data=[acl.id for acl in ACL.query.all()])
+        with self.Session() as session:
+            acls = session.scalars(ACL.select(session.user_or_token)).all()
+            return self.success(data=[acl.id for acl in acls])
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply skyportal/handlers/api/acls.py]: yup
